src/measure_theory/measure_space.lean

```diff
@@ -2171,15 +2171,19 @@ begin
         have h_meas_t_inter_s : measurable_set (t ∩ s) :=
            h_meas_t.inter h_meas_s,
         repeat {rw measure_eq_inter_diff h_meas_t h_meas_s, rw set.diff_eq},
-        apply add_le_add _ _; rw add_apply,
-        { apply le_add_right _,
+        apply add_le_add _ _,
+        { exact ordered_add_comm_monoid.to_covariant_class_left ℝ≥0∞ },
+        { exact ordered_add_comm_monoid.to_covariant_class_right ℝ≥0∞ },
```

Comment:
Do you know why these type-classes have to be given explicitly? Does it help if you replace `apply` with `refine`?

Reply:
Actually you are right: using `refine` removes the need for supplying the typeclasses!